src/index.ts

```diff
@@ -391,3 +391,51 @@ export class FileDisk extends Disk {
 		await fs.fdatasync(this.fd);
 	}
 }
+
+export class BufferDisk extends Disk {
+	constructor(
+		private readonly buffer: Buffer,
+		readOnly: boolean = false,
+		recordWrites: boolean = false,
+		recordReads: boolean = false,
+		discardIsZero: boolean = true,
+	) {
+		super(readOnly, recordWrites, recordReads, discardIsZero);
+	}
+
+	protected async _getCapacity(): Promise<number> {
+		return this.buffer.length;
```

Comment:
Where is an instance of `Buffer` assigned to `this.buffer`?

Reply:
@pdcastro when an argument of the constructor of a class is preceded with `public`or `private`, it is automatically assigned. See https://github.com/balena-io-modules/file-disk/pull/19/files#diff-f41e9d04a45c83f3b6f6e630f10117feR397